main.js

```diff
@@ -0,0 +1,48 @@
+
+snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velocirapto"
+};
+
+var guestOfHonor = snakewaterMontana.paleontologist;
+var cleverGirl = snakewaterMontana.specimen;
+console.log (cleverGirl);
+
+nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+ var specimens = nicaraguanSpecimens 
+ var favoriteSpecimen = nicaraguanSpecimens.velociraptor;
+ console.log (velociraptor)
+ nicaragua=(annualBudget= 400000);
```

Comment:
you don't need to use `()` and you don't want to do multiple assignments in one row it's either `nicaragua= 400000` or `annualBudget=400000;`  but both attempts doesn't make any sense why you want to reassign an object to a numerical value ?? you can access the annualBudget located in the nicaragua object like this `nicaragua.annualBudget` and reassign that value